cd-menu/cd-menu.js

```diff
@@ -53,19 +54,18 @@ window.cdMenu = function (options) {
   }
 
   function showElement (el) {
-    el.style.display = 'block';
+    el.style.display = this.display || 'block';
     // TODO fix mobile nav to use classes so this isnt needed
-    el.setAttribute('style', 'display: block !important;');
+    el.setAttribute('style', 'display: ' + el.style.display + ' !important;');
   }
 
   function hideElement (el) {
-    el.style.display = 'block';
     // TODO fix mobile nav to use classes so this isnt needed
     el.setAttribute('style', 'display: none !important;');
   }
 
   function showLoginRegister () {
-    each(loginRegisters, showElement);
+    each(loginRegisters, showElement.bind({ display: 'flex' }));
```

Comment:
Seems a little weird...

I'd be inclined to have `function showElement (el, display) {` in the function, and replace this with